KroneMultimedia/SharedFastfile.rb

```diff
@@ -0,0 +1,289 @@
+lane :before_build do
+  clear_derived_data
+  increment_build_number({
+      build_number: latest_testflight_build_number+1
+  })
+  cocoapods
+  unlock_keychain
+end
+
+lane :badge do
+  badge(shield: ENV['CI_BUILD_REF_NAME'], no_badge: true)
+end
+desc "Run Unit/UI Tests"
+lane :snapshots do
+  tests
+  #### PLACE FOR AWS
+end
+
+desc "Kill Simulator"
+lane :kill_simulator do
+  sh "killall iOS Simulator &> /dev/null || true"
+end
+
+desc "Run Unit/UI Tests"
+lane :tests do
+  certificates
+
+  kill_simulator
+  snapshot( {
+      workspace: $self_workspace,
```

Comment:
Seems like the tabs are wrong here. You can use `rubocop -a` and point to this file to fix it :+1: